runtime/src/configs/runtime_params.rs

```diff
@@ -92,21 +92,28 @@ pub mod dynamic_params {
 
         #[codec(index = 11)]
         #[allow(non_upper_case_globals)]
+        /// 0-size bucket fixed rate payment stream.
+        ///
+        /// Determining the rate to add to a fixed payment stream between an MSP and a user.
+        pub static ZeroSizeBucketFixedRate: Balance = 1;
```

Comment:
> We'd probably want to make this higher. At least leave a TODO to set this to a more reasonable value, or (better) set a reasonable value.

should we just set this to `1 * UNIT`? since dot fluctuates, how do we ensure that the price remains stable without relying on some price oracle? 

> Also, only use runtime parameters if you might need to change them later (like in tests). Because there is an overhead to using them.

what is the overhead?

Reply:
I just used the same `48 * PICOUNIT` used for `MostlyStablePrice`.